soc/arm/microchip_mec/mec1501/soc.c

```diff
@@ -136,6 +136,11 @@ static int soc_init(struct device *dev)
 	isave = __get_PRIMASK();
 	__disable_irq();
 
+	/* Hack: disable Cortex-M Serial Wire Viewer, only use
+	 * the two SWD pin (TMS & TCLK)
+	 */
+	ECS_REGS->DEBUG_CTRL = 0x05;
```

Comment:
We currently have this in the boards/arm boards/arm/mec15xxevb_assy6853/pinmux.c . We put it there to emphasize that we are releasing TDI/TDO. If you want you can set 0x05 there.

Reply:
Done